integration-test/global-setup.ts

```diff
@@ -0,0 +1,10 @@
+import dotenv from 'dotenv';
+
+export default async function globalSetup() {
+  const environment = process.env.NODE_ENV || 'local';
+
+  const envFile = environment === 'dev' ? '.env.dev' : '.env.local';
```

Comment:
@nora-weisser have to keep this file, because it is nice to keep different env setup, so we don't need to send the params, we can just setup locally our .env files `.env-local`, `.env-prod` and etc.